data/management/commands/import_issues_data.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand
+
+from data.issues import fetch_issues, import_issue
+
+
+class Command(BaseCommand):
+    help = 'Import issues opened by newcomers'
+
+    COLLECTIONS = staticmethod(fetch_issues)
```

Comment:
this doesnt make sense in this context.

It was only appropriate for OpenHub app commands, where there were many commands which were all identical.  And it would only have made sense if you had used inheritence, which you didnt.

It makes no sense at all when the `handle()` are not exactly the same. (which is where inheritance could be used, but you are not doing that).